File: OmniCRM_Web/Models/TargetMaster.cs
﻿using System;
using System.Collections.Generic;

// Code scaffolded by EF Core assumes nullable reference types (NRTs) are not used or disabled.
// If you have enabled NRTs for your project, then un-comment the following line:
// #nullable disable

namespace OmniCRM_Web.Models
{
    public partial class TargetMaster
    {
        public int TagetId { get; set; }
        public Guid TelecallerId { get; set; }
        public DateTime MonthYear { get; set; }
        public int TargetWeek1 { get; set; }
        public int TargetWeek2 { get; set; }
        public int TargetWeek3 { get; set; }
        public int TargetWeek4 { get; set; }
        public int TargetWeek5 { get; set; }
        public int TargetWeek6 { get; set; }

        public virtual UserMaster Telecaller { get; set; }
    }
}

File: OmniCRM_Web/ClientApp/src/app/models/target-master.ts
import { UserMaster } from "./user-master";

export class TargetMaster {
  tagetId: number;
  telecallerId: any;
  monthYear: Date;
  targetWeek1: number;
  targetWeek2: number;
  targetWeek3: number;
  targetWeek4: number;
  targetWeek5: number;
  targetWeek6: number;

  telecallerName: string;

  achieveWeek1: number;
  achieveWeek2: number;
  achieveWeek3: number;
  achieveWeek4: number;
  achieveWeek5: number;
  achieveWeek6: number;

}

export class TargetMatrix {
  header: string[] = [];
  rowDataTargetMaster: TargetMaster[] = [];
}

